config/packages/framework.yaml

```diff
@@ -6,3 +6,10 @@ framework:
         fallback: ['en']
     php_errors:
         log: true
+    serializer:
+        name_converter: 'serializer.name_converter.camel_case_to_snake_case'
+    cache:
```

Comment:
This is redundant - it's configuration for aggregates cache which we are not suggesting to have enabled in skeleton application